app/Config/Filters.php

```diff
@@ -49,7 +49,11 @@ class Filters extends BaseConfig
      * particular HTTP method (GET, POST, etc.).
      *
      * Example:
-     * 'post' => ['csrf', 'throttle']
+     * 'post' => ['foo', 'bar']
+     *
+     * If you use this, you should disable auto-routing. Because auto-routing
+     * permits any HTTP method to access a controller. Accessing the controller
```

Comment:
```suggestion
     * If you use this, you should disable auto-routing because auto-routing
     * permits any HTTP method to access a controller. Accessing the controller
```

Reply:
What's wrong with the period?

Reply:
The "because auto-routing..." is a subordinate clause. It doesn't have a meaning of its own.
https://www.grammarly.com/blog/subordinate-clause/